tests/test_factory/test_factory_db/invenio_accounts.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from invenio_accounts.models import User
+
+from .base import TestBaseModel
+
+
+class TestUser(TestBaseModel):
+    """
+    Create User instances.
+
+    Basic usage:
+        >>> factory = TestUser.create_from_kwargs(email='foo@bar.com')
+        >>> factory.user
+        <User (transient 4661300240)>
+        >>> factory.user.email
+        'foo@bar.com'
```

Comment:
You can use `Examples:` for this, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#docstring-sections.

Reply:
done